oauth2/oauth2-jose/spring-security-oauth2-jose.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
 	compile springCoreDependency
 	compile 'com.nimbusds:nimbus-jose-jwt'
 
+	optional 'com.nimbusds:oauth2-oidc-sdk'
```

Comment:
This dependency does not make sense. However, this will be removed when moving `OpenIdConfigurationJwtDecoderProvider` to the config module

Reply:
For the record, @jgrandja and I discussed, and this will stay here in `oauth2-jose` since `JwtDecoder`, the security component being instantiated by `JwtDecoders`, is in `oauth2-jose`.